src/Program.cs

```diff
@@ -0,0 +1,150 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+
+namespace TpmAgent;
+
+public class Program
```

Comment:
I would like the agent to use the Semantic Kernel Process Framework https://learn.microsoft.com/en-us/semantic-kernel/frameworks/process/process-framework

As an input it should take the content of a github issue.  

It should have a tool so that it can take action and comment on the issue